.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could we gitignore `.swiftpm`? I know [there are different opinions on this](https://stackoverflow.com/questions/65754188/should-i-add-swiftpm-to-gitignore), but as long as we don't need shared schemes or the like, I think this would be nice to avoid a bit of noise.